install.sh

```diff
@@ -16,6 +16,8 @@ if pip list | grep api-validator-python; then
 fi
 
 pip install --requirement requirements.txt
-python manage.py db upgrade
+if [ -z "$PROFILES_SKIP_DB" ]; then
+    python manage.py db upgrade
```

Comment:
Probably better to separate this as a step generally. Installing the dependencies shouldn’t require a functioning (ie configured) app.

Reply:
Yes, eventually these will be two different scripts. I think one run at container creation time, and the other to be run by the `cli` container as a custom entrypoint where needed (during database setup).